public_dropin_environments/r_lang/env_info.json

```diff
@@ -3,7 +3,7 @@
   "name": "[DataRobot] R 4.4.3 Drop-In",
   "description": "This template environment can be used to create artifact-only R custom models that use the caret library. Your custom model archive need only contain your model artifacts if you use the environment correctly.",
   "programmingLanguage": "r",
-  "environmentVersionId": "680a835568a555e652af1f9a",
+  "environmentVersionId": "6810fa73791e1c002d2b2645",
```

Comment:
I guess this update is typo?

Reply:
I dont't know, some Harness job did that commit to reconcile dependencies.